packages/@sanity/types/src/schema/types.ts

```diff
@@ -106,6 +106,7 @@ export interface ConditionalPropertyCallbackContext {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   value: any
   currentUser: Omit<CurrentUser, 'role'> | null
+  workspace: any
```

Comment:
> Also the current workspace object is so big (it includes schema, theme, and more) that it might not be a good idea to include all of it – or some trimmed-down version.

Agree with this. I think this trimmed-down version should be defined here instead of `any` (since this is the user-facing API).

What information about the workspace would be the most useful to include here?